libraries/ESP8266WiFi/examples/WiFiHTTPSServer/WiFiHTTPSServer.ino

```diff
@@ -0,0 +1,173 @@
+/*
+ *  This sketch demonstrates how to set up a simple HTTP-like server using HTTPS encryption.
+ *  A sample self-signed certificate is included.  For your own application, please be sure
+ *  to GENERATE YOUR OWN AND REPLACE.
```

Comment:
Adding some instructions on how to do that would be nice.

Also, same thing with public domain / CC0 license.

Reply:
Done (see above)